weblate/trans/util.py

```diff
@@ -268,7 +268,10 @@ def path_separator(path: str) -> str:
 
 def sort_unicode(choices, key):
     """Unicode aware sorting if available."""
-    return sorted(choices, key=lambda tup: locale.strxfrm(key(tup)))
+    try:
+        return sorted(choices, key=lambda tup: locale.strxfrm(key(tup)))
+    except OSError:
+        return sorted(choices, key=key)
```

Comment:
I don't like this approach, as it potentially hides errors from other code as well (especially the `key` callback). Probably there should be a function instead of lambda and it that it would be possible to catch `OSError` from `strxfrm` only.